src/middlewared/middlewared/schema.py

```diff
@@ -41,8 +41,8 @@ def __init__(self, *args, **kwargs):
         self.enum = kwargs.pop('enum', None)
         super(EnumMixin, self).__init__(*args, **kwargs)
 
-    def clean(self, value):
-        value = super().clean(value)
+    def clean(self, value, old_value=NOT_PROVIDED):
```

Comment:
Can we find another way to implement this without messing with the `Attribute` interface?